tests/e2e/helpers.ts

```diff
@@ -21,10 +21,17 @@ export async function login(
 }
 
 export function permissions(page: Page) {
-  const clear = async () => {
+  const clear = async (email = "permissions@example.com") => {
```

Comment:
Never have null variables as the first / or only one. Rather pass the default value.